modules/vcf/templates/annotate.sh

```diff
@@ -34,6 +34,57 @@ annot_sv() {
   fi
 }
 
+gado() {
+  if [ "!{areProbandHpoIdsIndentical}" ] && [ -n "!{gadoHpoIds}" ];
+  then
+    echo "RUN GADO RUN!"
+    gado_process
+    gado_prioritize
+  else
+    if [ "!{areProbandHpoIdsIndentical}" ];
+    then
+      echo "WARNING: HPO terms for proband(s) differ within project, skipping GADO!"
```

Comment:
1) log to stderr, instead of stdout? `>&2 echo "...."`
2) differ within project --> differ within sample sheet